notebooks/test_distributions.ipynb

```diff
@@ -0,0 +1,975 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    passing_df = pd.read_csv('../data/processed/passing_test.csv')
</pre><p>Can we use the data from the previous data collection step? Or if we dont want to store data locally, and if it doesnt make the notebook long, maybe we can store in-memory and do it in a single notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/optimal-stopping-point-tool/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Out of all the tests, we collect from step 1, I think we can leave out about 20% to evaluate our approach on and fit the rest to the curves. WDYT?</p>

Reply:
sure, that makes sense. We can combine this, and I can append this to the same notebook. 
And can split 80-20 and use that for validation

Reply:
The new updates are making the notebook super long and kernel is also dying, so limiting this to two notebooks atm. We can definitely make it one notebook once we are pass the experimenting phase.